examples/mini-hlint/mini-hlint.cabal

```diff
@@ -0,0 +1,21 @@
+name:                mini-hlint
+version:             0.1.0.0
+license:             BSD3
+x-license:           BSD-3-Clause OR Apache-2.0
+license-file:        ../../ghc-lib-gen/LICENSE
+author:              Shayne Fletcher (shayne.fletcher@digitalasset.com)
+maintainer:          Shayne Fletcher (shayne.fletcher@digitalasset.com)
+copyright:           Digital Asset 2018-2019
+build-type:          Simple
+cabal-version:       >=1.10
+
+executable mini-hlint
+  main-is:             Main.hs
+  build-depends:       base >=4.11
+                     , extra
+                     , ghc-lib
+                     , time
+  default-language:    Haskell2010
+  default-extensions:  PackageImports
```

Comment:
I'd suggest putting this in the file rather than here.

Reply:
Ok